site/en/docs/devtools/console/reference/index.md

```diff
@@ -87,6 +87,38 @@ messages][15] for a hands-on walkthrough.
 [Open Console Settings][16] and disable **Group similar** to disable the Console's default message
 grouping behavior. See [Log XHR and Fetch requests][17] for an example.
 
+### View stack traces {: #view-stack-traces }
+
+The **Console** automatically captures stack traces for errors and warnings. A stack trace is a history of function calls (frames) that led to the error or warning. The **Console** shows them in reverse order: the latest frame is at the top.
+
+To view a stack trace, click the {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/MaSHbLXzcTFbxhx9K8hX.svg", alt="Expand.", width="24", height="24" %} expand icon next to an error or warning.
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/TINzuAQUw7Ug3g2pOljm.png", alt="Stack traces.", width="800", height="401" %}
+
+#### View async stack traces {: #async-stack-traces }
+
+If supported by the framework you are using, DevTools can trace async operations by linking both parts of the async code together.
+
+In this case, the stack trace shows the "full story" of the async operation.
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/hyeBPwNtQQ933gjh7Sp7.png", alt="Async stack trace.", width="800", height="861" %}
+
+{% Aside 'gotchas' %}
+DevTools implements this "Async Stack Tagging" feature based on the `console.createTask()` API method. It is up to frameworks to implement the API. For example, [Angular supports this feature](/blog/devtools-better-angular-debugging/#the-async-stack-tagging-api-in-angular).
+{% endAside %}
+
+#### Show known third-party frames in stack traces {: #show-third-party }
+
+By default, the **Console** hides from stack traces the third-party frames from sources generated by bundlers (for example, webpack) or frameworks (for example, Angular).
```

Comment:
This requires that the source map generators include the `x_google_ignoreList` field, otherwise this wouldn't work out of the box.

I'd reword this to "When the source maps include the `x_google_ignoreList` field, the **Console** hides from stack traces the third-party frames from sources generated by bundlers (for example, webpack) or frameworks (for example, Angular) by default."